src/main/java/net/earthcomputer/clientcommands/mixin/MixinItemGroup.java

```diff
@@ -0,0 +1,43 @@
+package net.earthcomputer.clientcommands.mixin;
+
+import net.earthcomputer.clientcommands.interfaces.IItemGroup;
+import net.minecraft.item.ItemGroup;
+import net.minecraft.text.LiteralText;
+import net.minecraft.text.Text;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.Arrays;
+
+@Mixin(ItemGroup.class)
+public abstract class MixinItemGroup implements IItemGroup {
+
+    @Shadow @Mutable @Final public static ItemGroup[] GROUPS;
+
+    @Shadow @Mutable @Final private Text translationKey;
+
+    @Override
+    public void shrink() {
+        ItemGroup[] tempGroups = GROUPS;
+        tempGroups = Arrays.stream(tempGroups).filter(itemGroup -> !itemGroup.getName().startsWith("clientcommands.")).toArray(ItemGroup[]::new);
```

Comment:
`toArray(Supplier)` is not a Java 8 method

Reply:
Yes it is [(see the API documentation)](https://docs.oracle.com/javase/8/docs/api/java/util/stream/Stream.html#toArray-java.util.function.IntFunction-).

Reply:
Oh right, I forgot this was a stream rather than a collection